DevEdu.API/DevEdu.API/Controllers/User_GroupController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DevEdu.API.Models.InputModels;
+using DevEdu.DAL.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DevEdu.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class User_GroupController : Controller

```

Comment:
we don't need a separate controller for this. it could be a part of GroupController
the same works for repository